docs/source-app/index.rst

```diff
@@ -226,7 +226,8 @@ Keep Learning
    Extend an existing app <workflows/extend_app>
    Publish a Lightning component <workflows/build_lightning_component/publish_a_component>
    Run a server within a Lightning App <workflows/add_server/index>
-   Run an App on the cloud <workflows/run_app_on_cloud/index>
+   Run an app on the cloud <workflows/run_app_on_cloud/index>
+   Run Apps on your cloud account (BYOC) <workflows/byoc/index>
```

Comment:
The name "app" has a different case. Is there a reason?

Reply:
Nope, the first one is wrong. Good catch.